integrations/purchase/nordic-api-gateway.js

```diff
@@ -43,8 +41,8 @@ const agent = request.agent();
 agent.set('X-Client-Id', env.NAG_CLIENT_ID).set('X-Client-Secret', env.NAG_CLIENT_SECRET).set('Accept-Language', 'en');
 
 const NAG_CATEGORY = {
-  Supermarket: { purchaseType: PURCHASE_CATEGORY_FOOD_SUPERMARKET, activityType: ACTIVITY_TYPE_PURCHASE },
-  'Remodeling & Repair': { purchaseType: PURCHASE_CATEGORY_STORE_DEPARTMENT, activityType: ACTIVITY_TYPE_PURCHASE },
+  Supermarket: { purchaseType: PURCHASE_CATEGORY_STORE_FOOD, activityType: ACTIVITY_TYPE_PURCHASE },
+  'Remodeling & Repair': { purchaseType: PURCHASE_CATEGORY_STORE_CLOTHING, activityType: ACTIVITY_TYPE_PURCHASE },
```

Comment:
I would rather use PURCHASE_CATEGORY_STORE_HARDWARE for this category

Reply:
done